language/stdlib/modules/Diem.move

```diff
@@ -622,16 +715,34 @@ module Diem {
     /// at `preburn_address` does not contain a pending burn request.
     public fun cancel_burn_with_capability<CoinType>(
```

Comment:
find preburn resource and then pass the .
just change borrow global (from mutable ref on stack). then later destruction of preburn